pkg/arvo/sys/zuse.hoon

```diff
@@ -5094,9 +5094,29 @@
   ::
   |=  compare=$-([key key] ?)
   |%
```

Comment:
You'll need to put a fast hint on the whole core as well. This should work:

```
~%  %ordered-map  ..part  ~
|%
```

Reply:
I still get parent not found, but I'm not sure if that's expected or not.

Reply:
For which label; what's an example of the output? (Also, you might to `:goad %force` or something to ensure everything's rebuilt.)

Reply:
Once you've been messing around with the stuff, you really need to make a pill and boot from that to get a clean test.

I forgot that the %zuse core has a hint now, so we could try going straight to the right way. I think that would look like this:

```

++  ordered-map
  ~/  %ordered-map
  |*  [key=mold val=mold]
  =>  |%
      +$  item  [key=key val=val]
      --
  ::  +compare: item comparator for horizontal order
  ::
  ~%  %comp  +>+  ~
  |=  compare=$-([key key] ?)
  ~%  %core    +  ~
  |%
```

/cc @frodwith

Reply:
Booting from a solid pill formed with the changes you just specified, I still receive the parent not found:
<img width="683" alt="Screen Shot 2021-04-20 at 3 01 20 PM" src="https://user-images.githubusercontent.com/1377557/115456912-588cb380-a1e9-11eb-8294-c4df63161e1f.png">